pkg/artifactory/datasource/datasource_artifactory_group.go

```diff
@@ -0,0 +1,47 @@
+package datasource
+
+import (
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jfrog/terraform-provider-artifactory/v6/pkg/artifactory/resource/security"
+	"github.com/jfrog/terraform-provider-shared/packer"
+	"github.com/jfrog/terraform-provider-shared/predicate"
+	"github.com/jfrog/terraform-provider-shared/util"
+	"golang.org/x/net/context"
```

Comment:
Sort and group imports. @danielmkn can help you configure IntelliJ to sort these correctly.